bsp/avr32/at32uc3a0256/Kconfig

```diff
@@ -132,5 +132,25 @@ menu "On-chip Peripheral Drivers"
                     default 1 if BSP_UART1_RX_PIN_96
             endif
         endif
+
+    menuconfig BSP_USING_SOFT_I2C
+        bool "Enable I2C BUS (software simulation)"
+        default y
```

Comment:
except pin(GPIO) and serial, other devices should be n by default

Reply:
Dear @mysterywolf, Greetings of the day!

Thank you for your response. I have a few questions:

> except pin(GPIO) and serial, other devices should be n by default

I didn't understand this entirely. I can force "Enable I2C BUS" to
'n' by default. But in this case, it it the responsibility of a user
(of Mizar32) to manually configure OR enable the "Enable I2C BUS" in
order to use the LCD OR other I2C functions?

I assume that this approach will be rather tricky for a new user
trying to evaluate Mizar32 (OR other AVR32 BSPs).

Second: About RT-Thread packages: Is it a valid practice to enable
packages (by default) for any BSP? Say for example, I want to enable
the 'vi' text editor package by default for Mizar32 (which I am sure
will show up as configuration macros in rtconfig.h): is this permitted
within the RT-Thread code guidelines?

The Mizar32-A platform has 512 KB of flash an external SDRAM
controller. It will be nice for the default RT-Thread's Mizar32 build
to support on-board peripherals by default. OR am I getting something
wrong? Many thanks @mysterywolf.

Reply:
Dear @mysterywolf, Greetings!

> Hi, sorry for the late reply, please follow this link
> https://github.com/RT-Thread/rt-thread/blob/master/bsp/stm32/stm32l475-atk-pandora/board/Kconfig#L98

Many thanks for your message. Thank you for your
suggestion. Perfect. I will use this as a reference for my hacks
around LCD.

Second: As suggested by you, I have (by default) disabled software
i2c within Kconfig. Can you please check if this is acceptable?

Reply:
Thanks, this is correct. Only one of UART for console and GPIO should be default by yes, otherwise should default by n.